README.md

```diff
@@ -1,15 +1,41 @@
 # Cerbos Python Demo
 
-Run the following command to try out the demo:
+<img src="https://github.com/cerbos/cerbos/blob/main/docs/supplemental-ui/logo.png?raw=true" alt="Cerbos" width="87" height="100"/>
 
+This project `demo-python` will help you get familiar with using [Cerbos](https://github.com/cerbos/cerbos) in your Python applications.
+
+It's an app that follows access control regulation of a company employee `Harry` in process from submitting it, to getting his holiday request approved. Simply put, `demo-python` shows what are `Harry` and his work colleagues allowed to do with his `holiday request`.
+
+## Requirements
+- Minimum Python version supported by this demo is 3.10
+- Your Cerbos server is up and running (via either a [binary](https://docs.cerbos.dev/cerbos/latest/installation/binary.html), [container](https://docs.cerbos.dev/cerbos/latest/installation/container.html) or [helm](https://docs.cerbos.dev/cerbos/latest/installation/helm.html))
+
+⚠️ If you're completely new to Cerbos make sure to check the _[How it works](https://cerbos.dev/how-it-works)_ videos!  ⚠️
+
+
+## Run the demo
+
+Make sure that the `console` / `terminal` tab you're using is positioned in the root of this `demo-python` repository.
+You can do that by using the basic `cd` command which we will not be disecting now, but feel free to check how `cd` works in [Windows](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cd) and [Linux/Unix](https://www.javatpoint.com/linux-cd#:~:text=Linux%20cd%20command%20is%20used,commands%20in%20the%20Linux%20terminal) environments.
```

Comment:
I don't think we need to go into this level of detail. Our target audience is technical users who are familiar with these basic things.

Reply:
Hmm, I agree with you on this one. I had plenty of experiences where people were asking silly things but it takes more space than it adds value, will remove it.